tools/scripts-config-react-webpack/config/webpack.config.js

```diff
@@ -334,10 +336,9 @@ module.exports = ({ getUserConfig, mode }) => {
 						test: useTypescript ? /\.(js|ts|tsx)$/ : /\.js$/,
 						exclude: /node_modules/,
 						use: [
-							!process.env.NO_CACHE_LOADER && { loader: 'cache-loader' },
```

Comment:
Do you know if this was used ? `NO_CACHE_LOADER `
If so it's gonna break

Reply:
I have added that during CDN because of it was not possible for me to understand the cache key.
here we have fixed the cache key so I believe we will not having cache issue anymore.
I think we can remove it from everywhere and avoid confusion. WDYT ?